app/main.py

```diff
@@ -1,4 +1,5 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
+    validate(cat_age, dog_age)
```

Comment:
you are prohibited to change functions you are gonna test. the idea of tests is that somebody created some function and you test it regardless of it works correctly or not. your mission is just to test it with different sets of data